bundles/org.openhab.binding.deconz/README.md

```diff
@@ -319,12 +329,12 @@ end
 
 :::
 
-::: tab DSL
+::: tab JRuby
 
- ```java
- val deconzActions = getActions("deconz", "deconz:lightgroup:00212E040ED9:5");
- var retVal = deconzActions.createScene("TestScene");
- deconzActions.storeScene(retVal.get("newSceneId"));
+ ```ruby
+ deconz_thing = things["deconz:lightgroup:00212E040ED9:5"]
+ retval = deconz_thing.create_scene("TestScene")
```

Comment:
Just to be sure: So the method name `createScene` is automatically found from `create_scene`?

Reply:
Yes. The original `createScene` is still available, so `deconz_thing.createScene(.....)` can be used. But JRuby automatically creates an alias using Ruby's prefered snake case naming convention for all the camelCase methods.